components/button/button-add.js

```diff
@@ -24,7 +43,8 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 			 * When true, show the button with icon and visible text. When false, only show icon.
 			 * @type {boolean}
 			 */
-			textVisible: { type: Boolean, reflect: true, attribute: 'text-visible' }
+			textVisible: { type: Boolean, reflect: true, attribute: 'text-visible' },
+			_hasFocusOrHover: { state: true }
```

Comment:
This was added to pass the hover/focus state down to the +. I think it seems to be working as expected for focus-visible.